content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md

```diff
@@ -0,0 +1,125 @@
+
+21st Century IDEA states that new and updated executive agency websites should be “designed around user needs with data-driven analysis influencing management and development decisions.” It also says that we should, “continually test [our] website[s], web-based form[s], web-based application[s], or digital service[s] to ensure that user needs are addressed.” ([P.L. 115-336](https://www.congress.gov/bill/115th-congress/house-bill/5759/text) &sect;3(A)6). 
+
+This requirement really means that we need to [listen](https://designsystem.digital.gov/design-principles/#listen) to our customers and learn from what we hear—from the site’s creation through every update. We should use feedback to drive continuous improvement, making the site a little bit better with every update, and ensuring it meets customer expectations. So, what, exactly, does it mean to listen, and how do we design to listen? 
+
+[W. Edwards Deming](https://www8.gsb.columbia.edu/deming/about/history), whose theories on quality control sparked the renewal of Japan's economy following World War II and launched the total quality management movement, encouraged asking the **right questions**. 
+
+What are some of the **right questions** that we can ask to [discover](https://methods.18f.gov/discover/) what our customers want and need? 
+
+## Four Questions to Help Discover What Customers Want and Need 
+
+Consider the following four questions, based on the U.S. Web Design System’s [Listen design principle](https://designsystem.digital.gov/design-principles/#listen). 
+
+### Q1. Who are my customers? 
+
+Like any business or storefront, government websites have key, target audiences for our information and customers who want to use our services. 
+
+Use common profiles or [personas](https://methods.18f.gov/decide/personas/) to better understand customer needs, and improve your site to meet those needs. For example, [Code.gov uses four personas](https://github.com/GSA/code-gov/tree/master/docs/UX) to inform their content and site structure. 
+
+Also, follow these [tips for starting your customer experience journey](https://www.performance.gov/tips-for-starting-your-customer-experience-journey/). These are low-cost, low-lift actions you can take to develop a customer profile. 
+
+### Q2. What tools can I use to gather feedback? 
+
+The good news is that there are several tools available to gather feedback. Here are a few tools to help you get started: 
+
+- **[Digital Analytics Program (DAP)](https://digital.gov/services/dap/)**  - An analytics tool for measuring digital services in the federal government. It also powers [analytics.USA.gov](http://analytics.USA.gov), which offers a quick window into your website. (Note, [M-17-06 requires DAP](https://policy.cio.gov/web-policy/analytics) on all federal executive branch public-facing websites.)
```

Comment:
Update DAP URL from https://digital.gov/services/dap/ to https://digital.gov/guides/dap/.

Reply:
I did that on purpose; it's the real URL.. eventually, we will get back to a simple, single page under `/services/` which will be sep. from the group of pages in the `/guides/` directory.